AlphaWallet/Tokens/ViewControllers/TokenInstanceActionViewController.swift

```diff
@@ -133,7 +133,15 @@ class TokenInstanceActionViewController: UIViewController, TokenVerifiableStatus
         //TODO this will only contain values that has been resolved and might not refresh properly when the values are 1st resolved or updated
         //TODO rename this. Not actually `existingAttributeValues`, but token attributes
         let existingAttributeValues = tokenHolder.values
-        let cardLevelAttributeValues = action.attributes.resolve(withTokenIdOrEvent: tokenHolder.tokens[0].tokenIdOrEvent, userEntryValues: .init(), server: server, account: session.account, additionalValues: existingAttributeValues, localRefs: tokenScriptRendererView.localRefs)
```

Comment:
@oa-s Seem to affect TokenScript; give me some time to check it out. But it doesn't seem to match the PR title?

Reply:
pr remove static attributes storage  and uses one from asset definition store